### PR TITLE
fix(client): Improve client validation accessibility

### LINKED ISSUE #37377: Improve client validation accessibility on success state in Account Settings page
**Describe the bug**
Somewhat related to issue #37373. When error state or warning state is triggered by client validation, it is displayed with label color change and an error or warning message. But on success state, there is only label color change without success message.

According to WCAG, **only** changing color is not friendly to color blind or partially color blind users. There should be change in color and shape to accommodate those users.

**To Reproduce**
1. Go to http://localhost:8000/settings
2. Try to trigger success state in `Email Settings` (`Your Internet Presence` client validation is currently not working as described by related issue)
3. Refer to **Describe the bug**

**Expected behavior**
Add success message in addition to color change on success state, or add common semantic icon such as green check icon beside the label.

**Desktop (please complete the following information):**
 - OS: Windows 10 1809 Build 17763.832
 - Browser Latest Chrome
 - Version 77.0.3865.120 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by kishore-devaraj**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #37377

**Little Description:**
Added the check icon at the end of the FormControl Input field when the there is no error or warning state for more accessibility.

### CODE REVIEW

**Review by raisedadead — CHANGES_REQUESTED** [review]
I think this needs more work, the grayed out version of the check can be misleading, and it turning green is not very much noticeable.

Maybe the checkbox should not come up until the url has been validated. And once it does it should be green as it does in the final state.

Also we should add placeholder URLs to all these fields.

**Comment by kishore-devaraj** [comment]
@raisedadead I have updated the PR with the following:
1. No more grayed out version of check icon.
2. Check icon (Green) appears only when the validation is correct.
3. Dummy placeholder added for all the fields in internet presence.

**Comment by raisedadead** [comment]
Umm, you have a bunch of unrelated commits in here for some reason. You would need a rebase.

Reach out to [contributors chat room](https://gitter.im/FreeCodeCamp/Contributors) if you need help with that.

**Comment by kishore-devaraj** [comment]
@raisedadead Did a rebase and force pushed only commits required for this PR.

File: client/src/components/settings/Internet.js [hunk 1]
```diff
@@ -190,12 +204,12 @@ class InternetSettings extends Component {
               <ControlLabel>LinkedIn</ControlLabel>
               <FormControl
                 onChange={this.createHandleChange('linkedin')}
+                placeholder='https://www.linkedin.com/user-name'
```
Comment (raisedadead) [inline client/src/components/settings/Internet.js]:
```suggestion
                placeholder='https://www.linkedin.com/in/user-name'
```

File: client/src/components/settings/Internet.js [hunk 2]
```diff
@@ -218,12 +232,12 @@ class InternetSettings extends Component {
               <ControlLabel>Personal Website</ControlLabel>
               <FormControl
                 onChange={this.createHandleChange('website')}
+                placeholder='https://user-name.com'
```
Comment (raisedadead) [inline client/src/components/settings/Internet.js]:
```suggestion
                placeholder='https://example.com'
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
Nitpicks.

**Comment by kishore-devaraj** [comment]
@raisedadead Hope, everything is fine now.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**